tigerpath/majors_and_certificates/majors/PSY_2018.json

```diff
@@ -111,12 +116,28 @@
           "name": "Other Departmentals",
           "max_counted": 4,
           "min_needed": 4,
-          "explanation": "",
-          "course_list": [
-            "PSY 2*",
-            "PSY 3*",
-            "PSY 4*",
-            "PSY 5*"
+          "explanation": "Of the remaining departmental courses, all must be 200-level or higher and at least four must be 300-level or higher. Students can take up to two cognate courses from other departments, approved by the Psychology Departmental Representative, to count as departmental courses.",

```

Comment:
Something's not quite right here. If at least 4 of 4 courses muse be 300-level or higher, that's all of them. What am I missing?

Reply:
You're right I think I thought of PSY 300 as part of the 300-level courses but it shouldn't be.